osu.Server.Spectator/Startup.cs

```diff
@@ -80,8 +86,21 @@ protected virtual void ConfigureAuthentication(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IHostApplicationLifetime lifetime)
         {
+            var shutdownManager = app.ApplicationServices.GetService<GracefulShutdownManager>();
+
+            Debug.Assert(shutdownManager != null);
```

Comment:
You can actually skip the assertion by using `GetRequiredService<T>()` as already done below:

```suggestion
            var shutdownManager = app.ApplicationServices.GetRequiredService<GracefulShutdownManager>();
```

Reply:
Code no longer exists 🎉